scripts/deploy/minimefactory.js

```diff
@@ -0,0 +1,32 @@
+const logDeploy = require('@aragon/os/scripts/helpers/deploy-logger')
```

Comment:
Not sure, but was the minimefactory previously being deployed as part of the templates' deployments?

Reply:
Actually I deploy it in the bases and then use the same one for all the templates as an argument, [like here](https://github.com/aragon/dao-templates/pull/137/files#diff-57bb31259c782ab757b5f3a565861cdbR28)